src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatDataBlock.tsx

```diff
@@ -0,0 +1,57 @@
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useKeyStatQuery from '@common/modules/find-statistics/hooks/useKeyStatQuery';
+import React, { ReactNode } from 'react';
+import KeyStat from '@common/modules/find-statistics/components/KeyStat';
+
+export interface KeyStatDataBlockProps {
```

Comment:
formatting for this interface looks off

Reply:
I removed the new lines. I think I added them in an attempt to make it more readable.